os/booting-on-packet.md

```diff
@@ -18,11 +18,12 @@ Once logged into the portal you will be able to click the 'Deploy' button and ch
 If you elect to use the API to provision machines on Packet you should consider using [one of our language libraries](https://www.packet.net/dev/) to code against. As an example, this is how you would launch a single Type 1 machine in a curl command. [Packet API Documentation](https://www.packet.net/dev/api/)
 
```

Comment:
Yes there is--I'll add that when I get a chance here. Thanks for the suggestion.